tests/system.py

```diff
@@ -2414,9 +2414,8 @@ def test_querying_data_w_timeout(self):
 
         query_job = Config.CLIENT.query(
             """
-            SELECT name, SUM(number) AS total_people
-            FROM `bigquery-public-data.usa_names.usa_1910_current`
-            GROUP BY name
+            SELECT COUNT(*)
```

Comment:
Not really associated with this sample, but the system test flaked, so I figured I'd swap out this query with one from go/bq-long-queries

Reply:
Sounds good.